cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -139,42 +165,46 @@ export default {
         return { content: contents[id], flipped: false };
       };
 
-      this.cards = Object.keys(contents).filter(_filter).map(_createCard);
+      this.credentials = Object.keys(contents).filter(_filter).map(_createCard);
 
-      console.log(`showing ${this.cards.length} credentials`);
+      console.log(`showing ${this.credentials.length} credentials`);
 
-      // Reading the values of the credentials and mapping it to the credential display data schemas
-      this.cjson.forEach((obj) => {
-        let flattened = {};
-        for (let credentialKey in this.cards) {
-          this.flatten(this.cards[credentialKey].content, flattened);
-          for (let credentialContent in flattened) {
-            if (
-              obj.credentialSubject.hasOwnProperty(credentialContent) &&
-              obj.schema === this.credDisplayName(this.cards[credentialKey].content)
-            ) {
-              obj.credentialSubject[credentialContent] = flattened[credentialContent];
-            }
+      Object.entries(this.credentials).forEach(([key]) => {
+        let manifestCredential;
+        let cType = this.filterType(this.credentials[key].content);
```

Comment:
If we don't plan to modify this variable, I would suggest making it a `const` - see https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75 for reference, for instance

Reply:
Unfortunately we have plan to modify this variable and using const dont work , i needed to use let.

Reply:
But you don't modify it anywhere